### PR TITLE
Add some framework tests

### PR DESCRIPTION
**Pull request by Jym77**
Add basic test for building pages out of React or Vue components.
Didn't manage to build any Angular component 😖

### CODE REVIEW

File: packages/alfa-angular/tsconfig.json [hunk 1]
```diff
@@ -1,4 +1,7 @@
 {
+  "compilerOptions": {
+      "experimentalDecorators": true
+  },
```
Comment (zsofitoth) [inline packages/alfa-angular/tsconfig.json:4]: Is this change necessary? I don't see other updates to the `angular.spec.tsx`

**Review by zsofitoth — APPROVED** [review]
Looks good to me 👍

File: packages/alfa-angular/tsconfig.json (hunk 1 above)
Comment (Jym77) [reply packages/alfa-angular/tsconfig.json]: Indeed, not needed until we actually get to write a test that needs it.
